services/gps2tsdb/gps.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python3
+
+# DBUS imports
+from gi.repository import GLib
+from dbus.mainloop.glib import DBusGMainLoop
+import dbus
+
+# Postgres imports
+import postgres
+
+# Environmental variable reading
+import os.environ
+
+
+# Process GPS data, called every time a fix is acquired
+def fix(*args):
+    # Data comes in as an array, extract the relevant data. See the DBUS section of `man 8 gpsd` for more information
+    time = float(args[0])
+    lat = float(args[3])
+    lng = float(args[4])
+
+    # Print it all out for debugging
+    print("Time: ", time, "\tLat: ", lat, "\tLng: ", lng)
+    print("Inserting lat and lng for timestamp", time)
+    # Insert into database using global db var
+    db.run("INSERT INTO gps (time, lat, lng) VALUES (to_timestamp(%s), %s, %s)", (time, lat, lng) )
```

Comment:
When we merge this, can we also open a issue noting that we aren't logging all of the GPS data yet.